terranova/commands/binds.py

```diff
@@ -338,9 +360,13 @@ def plan(
         exit_code = 1 if 1 in error_exit_codes else 2
         raise Exit(code=exit_code)
 
+    # Generate terranova plan
+    if out:
+        out.write_text(json.dumps(execution_plan))
+
 
 @click.command("apply")
-@click.argument("path", type=str, required=False)
+@click.argument("path_or_plan", type=str, required=False)
```

Comment:
I don't fully understand this.

How does this affect the CLI usage?

We are currently planning/applying each resource in a separate step in BK.

We actually cannot do plan or apply for all resources because the vault token expires.

Reply:
Will I still be able to do 

```shell
terranova apply repos/oblt-actions tfplan
```

?

Reply:
It's more something like this:
```
terranova plan repos/oblt-actions --out=plan.tnplan
terranova apply plan.tnplan
```
The execution plan already contains all group resources target by `repos/oblt-actions`.

Reply:
I see, so this actually needs to have the `.tnplan` extension.